rds.go

```diff
@@ -266,6 +311,18 @@ func NewRDSExporter(sess *session.Session, logger log.Logger) *RDSExporter {
 			[]string{"aws_region", "dbinstance_identifier"},
 			nil,
 		),
+		LogsStorage: prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, "", "rds_logsstorage"),
+			"The amount of storage consumed by log files (in bytes)",
+			[]string{"aws_region", "dbinstance_identifier"},
+			nil,
+		),
+		Logs: prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, "", "rds_logs"),
```

Comment:
how about rds_logs_total? _number is too generic, _count would be nice but this is not really a counter and therefore _count might be misleading. up to you @lechuk47

Reply:
I also considered `_total` but then got caught in the "_total_ what?" question. I agree `_number` is not very good too so here's a third proposal to make things more exciting...

```suggestion
			prometheus.BuildFQName(namespace, "", "rds_log_amount"),
```

Reply:
Ahh, the naming things and cache invalidations problems strike again.  
To me, `_count` fits better, but I agree that it could be misleading. I'll go with `_amount`.